### DISCUSSION TITLE
Orario docenti

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#758] [Idee e proposte]
Buonasera ,sarebbe possibile provvedere all'integrazione dell'orario dei docenti all'interno del registro?Lo chiedo perché più si va avanti e più io ,come altri docenti ,ci rendiamo conto che ,dover ogni volta consultare il formato cartaceo che tante volte si perde o comunque non si ha a portata di mano ,diventa difficile ,soprattutto per i docenti con più cattedre stabilire l'ora in cui firmare .

**Comment by trinko** [comment]
Al momento non mi è possibile apportare queste modifiche.
